feed.xml

```diff
@@ -12,12 +12,14 @@ layout: null
     <pubDate>{{ site.time | date_to_rfc822 }}</pubDate>
     <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
     <generator>Jekyll v{{ jekyll.version }}</generator>
-    {% for page in site.html_pages %}
+    {% assign sorted_pages = site.html_pages | sort:"last_modified_at" | reverse %}
+    {% for page in sorted_pages %}
       {% unless page.tag == 'omit' %}
         <item>
           <title>{{ page.title | xml_escape }}</title>
           <description>{{ page.content | xml_escape | strip_html | truncatewords: 75 }}</description>
           <link>{{ page.url | prepend: site.url }}</link>
+          <pubDate>{{ page.last_modified_at | date_to_rfc822 }}</pubDate>
```

Comment:
`<pubDate>` seems to be for `publication date`, but in fact it is not.
Are there tags available for update or modification date?

Reply:
Per RSS specification, the only tags that are available for the `<item>` element are `pubDate` and `<lastBuildDate>`.